gatsby-node.js

```diff
@@ -5,3 +5,53 @@
  */
 
 // You can delete this file if you're not using it
+
+const path = require('path');
+const { createFilePath } = require('gatsby-source-filesystem');
+
+exports.createPages = ({ actions, graphql}) => {
```

Comment:
Creates a query for getting blog posts and generates an html page on the site for the posts.